pyglmnet/pyglmnet.py

```diff
@@ -678,47 +688,100 @@ def fit_predict(self, X, y):
         """
         return self.fit(X, y).predict(X)
 
-    def score(self, y, yhat, ynull=None, method='deviance'):
+    def score(self, X, y):
         """Score the model.
 
         Parameters
         ----------
-        y : array, shape (n_samples, [n_classes])
+        X : array,
+            (n_samples, n_features)
             The true labels.
-        yhat : array, shape (n_samples, [n_classes])
+        y : array,
+            (n_samples, [n_classes])
             The estimated labels.
-        ynull : None | array, shape (n_samples, [n_classes])
-            The labels for the null model. Must be None if method is 'deviance'
-        method : str
-            One of 'pseudo_R2' or 'deviance'
+
+        Returns
+        -------
+        score: array | float
+            array when score is called by a list of estimators `glm.score()`
+            float when score is called by a sliced estimator `glm[0].score()`
+
+            Note that if you want compatibility with sciki-learn's
+            pipeline, cross_val_score, or GridSearchCV then you should
+            only pass sliced estimators:
+
+            from sklearn.grid_search import GridSearchCV
```

Comment:
This is awesome, thanks! Will keep in mind while doing a re-read of all docstrings